FreeRTOS/Demo/Posix_GCC/.vscode/settings.json

```diff
@@ -10,7 +10,15 @@
                 "MSYSTEM": "MSYS",
                 "CHERE_INVOKING": "1",
             }
+        },
+        "Cygwin": {
+            "path": "C:\\cygwin64\\bin\\bash.exe",
+            "args": ["--login", "-i"],
+            "env": {
+                "MSYSTEM": "Cygwin",
+                "CHERE_INVOKING": "1",
+            }
         }
     },
-    "terminal.integrated.defaultProfile.windows": "MSYS2",
+    "terminal.integrated.defaultProfile.windows": "Cygwin", //    "terminal.integrated.defaultProfile.windows": "MSYS2",
```

Comment:
Why this change? Would it break other MSYS2?

Reply:
Yes i believe so, Thanks for pointing out..
May be i will revert back this change..

Reply:
The main reason why I added cygwin is because I was getting error with MSYS2
Error was something like unable to find "sys/times.h" while compilation (I'm not sure if it was times.h or other header file).

But the Cygwin was perfectly compiling and working properly.

Reply:
Can you not select the default profile by running **Terminal: Select Default Profile** as described here - https://code.visualstudio.com/docs/terminal/profiles?

Reply:
Dumb question: I realize this would modify the default profile to use the Cygwin profile instead of the MSYS2 profile by default and may cause a failure for MSYS2, but wouldn't this defaulting to MSYS2 also impact Cygwin compilation?

If so maybe a comment here and/or an update to the README is required for this demo to select the correct profile.

Reply:
@kstribrnAmzn , Yes correct, but my intention is to keep the 'default profile' modification only to my local copy (I wont be uploading it).
@aggarg , Please let us know if we have any other solution.

Reply:
@joejoseph08 Do you plan to update as you suggested?

Reply:
@aggarg , Yes sir
I will do it End of Today itself...
Appologize for the late response..

Reply:
@aggarg , I've completed the revert of my modification.

Reply:
Merged your PR. Thank you for your contribution!

Reply:
> Merged your PR. Thank you for your contribution!

Thank you for your patient support too :)